openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Bots.spec.js

```diff
@@ -66,6 +66,11 @@ describe('Bots Page should work properly', () => {
     //Enter description
     cy.get(descriptionBox).type(description);
     interceptURL('POST', '/api/v1/bots', 'createBot');
+     //Click Password Generator
+    cy.get('[data-testid="password-generator"]')
+    .should('be.visible')
+    .click();
+  cy.wait(1000);
```

Comment:
It should wait on API response instead random time. Sync with @ShaileshParmar11 to get insight

Reply:
@chirag-madlani  Updated!